src/components/callout/examples/callout-caution.tsx

```diff
@@ -0,0 +1,16 @@
+import { Component, h } from '@stencil/core';
+
+/**
+ * Type: `caution`
+ */
+@Component({
+    tag: 'limel-example-callout-caution',
+    shadow: true,
+})
+export class CalloutCautionExample {
+    public render() {
+        return (
+            <limel-callout type="caution">I hope you read this.</limel-callout>
```

Comment:
😄 way better than Lorem Ipsum! Thanks

Reply:
I totally stole that from [remark-admonitions](https://github.com/elviswolcott/remark-admonitions#infima-docusaurus-v2), and yes, I really like it too! 👍